llvm/test/Transforms/LoopVectorize/RISCV/zext-drop-nneg-vplan.ll

```diff
@@ -0,0 +1,93 @@
+; RUN: opt -passes=loop-vectorize -mtriple=riscv64-unknown-linux-gnu -mattr=+v < %s -debug-only=loop-vectorize -disable-output 2>&1 | FileCheck %s
+; REQUIRES: asserts
+
+define void @test() {
```

Comment:
It should be possible to have the print test target independent in `llvm/test/Transforms/LoopVectorize/vplan-printing.ll` as testing printing doesn't need conditional blocks I think